tests/test_cli.py

```diff
@@ -155,40 +156,53 @@ def cmd(projection):
     assert result.output.splitlines() == ['geographic']
 
 
-def test_sequence(runner):
+@pytest.mark.filterwarnings("ignore")
+@pytest.mark.parametrize(
+    ("opt", "val"),
+    [
+        ("--sequence", True),
+        ("--no-sequence", False),
+        (None, cligj.__version__.startswith("1.0")),
```

Comment:
This test will fail if we up the version to 1.0 without making the change we advertised, something I've slipped up on before.